elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/orm/jpa/BasicEntityManagerProxy.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023, the original author or authors.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.spring.orm.jpa;
+
+import com.yahoo.elide.datastores.jpa.SupplierEntityManager;
+
+import org.springframework.orm.jpa.EntityManagerProxy;
+import jakarta.persistence.EntityManager;
+
+import java.util.function.Supplier;
+
+/**
+ * Basic EntityManagerProxy implementation.
+ */
+public class BasicEntityManagerProxy extends SupplierEntityManager implements EntityManagerProxy {
```

Comment:
What is the purpose of wrapping the entity manager with this proxy?  I don't see any direct calls to getTargetEntityManager.

Reply:
I did this more because of consistency as Spring creates their own proxies implementing the `EntityManagerProxy` and I'm not sure if it might be useful in future to get the underlying `EntityManager`.